tests/data_manager/test_netcdf_writer.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import xugrid as xu
+import numpy as np
+from pathlib import Path
```

Comment:
Ruff flagged this import as 'unused'. Could you remove it, please? After that you can merged the code to the dev branch